test/test_lowlevel.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+from pudb.lowlevel import detect_encoding, decode_lines
+from pudb.py3compat import PY3
+
+
+def test_detect_encoding_nocookie():
+    lines = ['Test Проверка']
```

Comment:
It's been awhile since I've dealt with this stuff so I can't remember if it is needed. The `u` does nothing in Python 3, so it would only be a question of the Python 2 behavior. I had thought that strings without `u` in Python 2 with non-ASCII characters are undefined, but perhaps the coding line at the top of this file fixes that. I'm also not clear, but if this is the specific thing that this function is testing, then this is fine.